SphericalShell/OpenMC_Input/OpenMC_Model/SS_Source_Tallies_Model.py

```diff
@@ -0,0 +1,41 @@
+import openmc
+import yaml
+
+file = open("OpenMC_SS_YAML.yaml")
+data = yaml.safe_load(file)

```

Comment:
you don't use this data in the end

Reply:
Accidentally left this in - it's been deleted now. I'm assuming the user will import the file when it's time to call these functions